package-lock.json

```diff
@@ -0,0 +1,1940 @@
+{
```

Comment:
I tried to avoid committing a package-lock as it increases the effort merging PR's. Would be happier about .git_ignore on the package-lock.

Reply:
I think it's not as bad as it once was, but sure it's no biggie